src/wrapper/jnienv.rs

```diff
@@ -1757,6 +1753,42 @@ impl<'a> JNIEnv<'a> {
         self.get_static_field_unchecked(class, (class, field, sig), ty)
     }
 
+    /// Set a static field. Requires a class lookup and a field id lookup internally.
+    pub fn set_static_field<'c, 'f, T, U>(&self, class: T, field: U, value: JValue) -> Result<()>
+    where
+        T: Desc<'a, JClass<'c>>,
+        U: Desc<'a, JStaticFieldID<'f>>,
+        //U: Into<JNIString>,
+    {
+        let class = class.lookup(self)?.into_inner();
+        let field = field.lookup(self)?.into_inner();
+
+        match value {
+            JValue::Object(v) => jni_unchecked!(
```

Comment:
What happened to `JavaType::Array()`?

Reply:
As far as I can see, there is no way to do something like `SetStaticArrayField`. I'm not a Java expert, so I can be missing something.

Reply:
That's correct: arrays are object references in Java, and are represented in JValue enum as JValue::Object (there is no JValue::Array). The C JNI API accepts them as an argument to `SetStaticObjectField`